### PR TITLE
[NCL-5010] Add user related logs

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added a note in the CHANGELOG.md for your change if user-facing?

### CODE REVIEW

**Comment by thescouser89** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

File: common/src/main/java/org/jboss/da/common/util/LoggerProducer.java [hunk 1]
```diff
@@ -16,4 +16,11 @@
     public Logger produceLogger(InjectionPoint ip) {
         return LoggerFactory.getLogger(ip.getMember().getDeclaringClass());
     }
+
+    @Produces
+    @UserLog
+    public Logger produceUserLogger() {
+        return LoggerFactory.getLogger("org.jboss.pnc.userlog.da");
```
Comment (matejonnet) [inline common/src/main/java/org/jboss/da/common/util/LoggerProducer.java]: Logger name have to be "org.jboss.pnc.\_userlog\_.da"

File: communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java [hunk 2]
```diff
@@ -83,6 +85,7 @@ public AproxConnectorImpl(Configuration configuration) {
         }
 
         try {
+            userLog.info("Retrieving versions for maven artifacts " + ga + " from " + query);
             log.info("Retrieving maven metadata for " + ga + " from " + query);
```
Comment (matejonnet) [inline communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java]: No need to duplicate the logs, user logs end up with others, only the logger name is different.

File: common/src/main/java/org/jboss/da/common/util/LoggerProducer.java (hunk 1 above)
Comment (janinko) [reply common/src/main/java/org/jboss/da/common/util/LoggerProducer.java]: JIRA understoot the underscores as formating for italics :/

**Pull request merged**